sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -180,6 +180,9 @@ export async function saveFakeApplicationDisbursements(
       livingWithParents: FormYesNoOptions.Yes,
       numberOfParents: 2,
     },
+    dmnValues: {
```

Comment:
Not a part of this PR but the `workflowData` set inside the application factory as it is seems to be wrong and not something to be followed.